docs/kb/api-version-4.md

```diff
@@ -1,48 +1,98 @@
+Upon logging in, a new field will be returned in the login response. 
+This will be your indication that you may no longer use version 3 (or any other previous version), and must now use the version 4 syntax.
+Here's what that will look like:
 
 Request:
 ```
 POST: https://pi.pardot.com/api/login/version/3
 message body: email=<email>&password=<password>&user_key=<user_key>
 ```
 
-Response:
+Response (Before transition):
+```
+<rsp stat="ok" version="1.0">
+  <api_key>1234abcd</api_key>
+</rsp>
+```
+
+Response (After transition):
 ```
 <rsp stat="ok" version="1.0">
   <api_key>1234abcd</api_key>
   <version>4</version>
 </rsp>
-```
\ No newline at end of file
+```
+
+*Note: Using version 3 to login will continue to work, even though all other endpoints will require version 4*
+
+## Request Path Changes
+
+You will need to modify your request to use the `/version/4` path. For example:
+
+Before transition:
+```
+GET: https://pi.pardot.com/api/prospect/version/3/read/id/1
+```
+After transition:
+```
+GET: https://pi.pardot.com/api/prospect/version/4/read/id/1
+```
+
+
+## Prospect API changes
+
+Most of the changes in version 4 occur around the prospect APIs. 
+When you begin using the version 4 APIs, your logic should know about these changes ([full documentation can be found here](/kb/api-version-4/prospects))
+
+* **New reference field:**  Prospects can now be referenced using the Salesforce CRM Identifier, known to the api as the `fid` field.
+This `fid` field refers to the Salesforce identifier applied to Leads or Contacts. 
+* **Create:** Prospects can still be created with a referenced `email` address.
+If you call the create api with the same email address multiple times, each call will create a new prospect.
+Previously, you would get an error.
+* **Read:** When making a read query, querying by `email` will return all prospects that share that email address.
+Querying by Pardot `id` or Salesforce `fid` will only return the one matching prospect.
+* **Update:** Prospects can no longer be updated when referenced via `email` alone. 
+This is because there may now be multiple prospects with the same email address.
+* **Upsert:** Upsert query by `email` will *always* create a new prospect.
+Upsert by Pardot `id` or Salesforce `fid` will update or create as needed.
+See [Upserting Prospects](/kb/api-version-4/prospects/#upserting-prospects) for more details.
+* **Delete:** Delete query must reference by Pardot `id` or Salesforce `fid`.
+* **Batch processing:**  Examples of batch processing can be found [here](/kb/api-version-4/prospects/#endpoints-for-batch-processing)
+
+## Opportunity API changes
+
+[Full documentation can be found here.](/kb/api-version-4/opportunities/)
+
+* **Create:** Creating an opportunity using a `prospect_email` reference must correspond to an existing prospect.
+If there are multiple prospects with that email address, the prospect with the most recent activity date will be used. 
```

Comment:
Can I use a prospect_id or prospect_fid?  If so, then might want to add "If you want to apply the opportunity to a specific prospect, then you must reference the prospect using `prospect_id` or `prospect_fid`."